apps/android/common/src/commonMain/resources/MR/el/strings.xml

```diff
@@ -42,7 +42,6 @@
     <string name="accept_connection_request__question">Αποδοχή αιτήματος σύνδεσης;</string>
     <string name="callstatus_accepted">αποδεκτή κλήση</string>
     <string name="network_enable_socks_info">Πρόσβαση στους διακομιστές μέσω SOCKS proxy στην πόρτα 9050; Ο διακομιστής μεσολάβησης (proxy server) πρέπει να είναι ενεργός πριν ενεργοποιηθεί αυτή η ρύθμιση.</string>
-    <string name="accept_requests">Αποδοχή αιτημάτων</string>
```

Comment:
string removed

Reply:
There is not such string key in base locale. so it should be removed. Unused